src/_guides/language/language-tour.md

```diff
@@ -505,6 +505,14 @@ var y = 1.1;
 var exponents = 1.42e5;
 ```
 
+You can also declare a variable as a num. If you do this, the variable
+can have both integer and double values.
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (declare-num)"?>
+```dart
+num x = 1; // x can have both int and double values
+```
```

Comment:
Would it be worth showing the variable containing a double? Maybe follow it with:
```dart
x += 2.5;
```